src/lib/style/components/_download-link.scss

```diff
@@ -0,0 +1,57 @@
+.Download-Link {
+  position: relative;
+  display: inline-block;
+  .icon-container {
```

Comment:
if we are to use bems - this class would be `Download-Link__icon container ` and then in the scss we'd have
```
.Download-Link {
  position: relative;
  display: inline-block;
 &__icon-container {
...
```